lockup-contract/src/lib.rs

```diff
@@ -133,70 +148,66 @@ impl LockupContract {
     ///    If it's used in addition to the vesting schedule, then the amount of tokens available to
     ///    transfer is subject to the minimum between vested tokens and released tokens.
     /// - `staking_pool_whitelist_account_id` - the Account ID of the staking pool whitelist contract.
+    ///    The version of the contract. It is a monotonically increasing number.
+    /// - `version` - Version of the lockup contract will be tracked by the veNEAR contract.
     #[init]
     pub fn new(
         owner_account_id: AccountId,
+        venear_account_id: AccountId,
         lockup_duration: WrappedDuration,
         lockup_timestamp: Option<WrappedTimestamp>,
-        transfers_information: TransfersInformation,
         release_duration: Option<WrappedDuration>,
```

Comment:
No more lockup_timestamp and release_duration, since they are triggered by the user.